index.js

```diff
@@ -69,6 +70,7 @@ function createDatDNS (datDnsOpts) {
     var ignoreCachedMiss = opts && opts.ignoreCachedMiss
     var noDnsOverHttps = opts && opts.noDnsOverHttps
     var noWellknownDat = opts && opts.noWellknownDat
+    var followRedirects = (opts && opts.followRedirects) || DEFAULT_FOLLOW_REDIRECTS
```

Comment:
Could we modify this so that `0` is allowed?

Reply:
added support & test in 7a488de - thanks for noticing.